tools/reviewer/local_server/service/code_review.proto

```diff
@@ -78,7 +78,8 @@ message Diff {
   string workspace = 8;
   int64 created_timestamp = 9;
   int64 modified_timestamp = 10;
-  repeated Thread thread = 13;
+  // Threads on a line.
+  repeated Thread line_thread = 13;
```

Comment:
After some more thought, I think we need an enum: https://github.com/google/startup-os/pull/323
Feel free to copy the changes into your PR.